package.json

```diff
@@ -35,15 +35,20 @@
     "tldjs": "^1.7.0"
   },
   "devDependencies": {
+    "autoprefixer": "^6.7.5",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
     "babel-preset-stage-1": "^6.16.0",
     "babelify": "^7.3.0",
     "browserify": "^13.1.1",
+    "css-modulesify": "^0.27.1",
     "gulp": "^3.9.1",
     "gulp-identity": "^0.1.0",
     "gulp-uglify": "^2.0.1",
     "loose-envify": "^1.3.0",
+    "postcss": "^5.2.15",
+    "postcss-cssnext": "^2.9.0",
+    "postcss-modules-values": "^1.2.2",
```

Comment:
Only `postcss-cssnext` is used directly, can the other two be removed without breaking it?